datetime/src/commonMain/kotlin/DateTimeComponents.kt

```diff
@@ -0,0 +1,63 @@
+/*
+ Copyright 2020 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.datetime
+
+import com.splendo.kaluga.datetime.datecomponents.Day
+import com.splendo.kaluga.datetime.datecomponents.Month
+import com.splendo.kaluga.datetime.datecomponents.Year
+
+data class DateTimeComponents(
```

Comment:
If you're gonna use DateComponents, you should also introduce TimeComponents imo

Reply:
now data Components includes time components (some of them))) but yes, good idea